Apps/WebClient/src/ClientApp/src/components/modal/AppTourComponent.vue

```diff
@@ -0,0 +1,202 @@
+﻿<script lang="ts">
+import { BCarousel } from "bootstrap-vue";
+import Vue from "vue";
+import { Component, Ref } from "vue-property-decorator";
+import { Getter } from "vuex-class";
+
+import type { WebClientConfiguration } from "@/models/configData";
+
+interface TourSlide {
+    // Title to be displayed for the slide.
+    title: string;
+    // Description to be displayed for the slide.
+    description?: string;
+    // Image or Gif to be displayed for the slide.
+    imageUri: string;
+    // Optional alt text for the image.
+    imageAlt?: string;
+}
+
+@Component
+export default class AppTourComponent extends Vue {
+    @Getter("webClient", { namespace: "config" })
+    webClient!: WebClientConfiguration;
+
+    @Ref("tourCarousel")
+    readonly tourCarousel!: BCarousel;
+
+    @Getter("isMobile")
+    isMobile!: boolean;
+
+    private mobileSlides: TourSlide[] = [
+        // TODO: Final configuration in AB#15392
+        {
+            title: "MB-Add a Quick Link",
+            description:
+                "Add a quick link to easily access a health record type from your home screen.",
+            imageUri: new URL(
+                "@/assets/images/tour/add-quicklink.png",
+                import.meta.url
+            ).href,
+            imageAlt: "First slide",
+        },
+        {
+            title: "MB-Filter Timeline Data",
+            description: "Follow the demo above to filter your timeline data.",
+            imageUri: new URL(
+                "@/assets/images/tour/filter-timeline.gif",
+                import.meta.url
+            ).href,
+            imageAlt: "Second slide",
+        },
+    ];
+
+    private desktopSlides: TourSlide[] = [
+        // TODO: Final configuration in AB#15392
+        {
+            title: "Add a Quick Link",
+            description:
+                "Add a quick link to easily access a health record type from your home screen.",
+            imageUri: new URL(
+                "@/assets/images/tour/add-quicklink.png",
+                import.meta.url
+            ).href,
+            imageAlt: "First slide",
+        },
+        {
+            title: "Filter Timeline Data",
+            description: "Follow the demo above to filter your timeline data.",
+            imageUri: new URL(
+                "@/assets/images/tour/filter-timeline.gif",
+                import.meta.url
+            ).href,
+            imageAlt: "Second slide",
+        },
+    ];
+
+    private slideIndex = 0;
+
+    private isVisible = false;
+
+    public showModal(): void {
+        this.isVisible = true;
+    }
+
+    public hideModal(): void {
+        this.isVisible = false;
+    }
```

Comment:
Should slideIndex be reset when the modal is hidden?

```suggestion
    public hideModal(): void {
        this.isVisible = false;
        this.slideIndex = 0;
    }
```